### ISSUE TITLE
Screen repainting doesn't work

### ISSUE THREAD

**Issue by tmcmh** [danoon2/Boxedwine#59]
I'm not sure how to clearly describe what I'm seeing (running Windows 10 with BoxedWIne using Windows 98 and Wine 5.0). When I run a program that requires any kind of scrolling, like a drop-down menu, the screen doesn't repaint correctly, instead it over-writes what's there with new data, but not in the right location. I'm using BoxedWine binaries downloaded from boxedwine.org.

I tried various configurations -- different Windows versions, poll rates etc to no avail. I'm primarily using BoxedWIne to play Stars! where the behavior is obvious but it's not unique to Stars!

**Comment by danoon2** [comment]
I've also noticed that strange scrolling behavior on other Win16 games, like Castle of the Winds.  I never looked into it much because there were always more pressing issues, but just for fun I tried Stars and Castle of the Winds in my newest branch where I'm working on Vulkan and Wine 6/7.  I saw that with the Wine 7 file system it seems to work fine now.

Here is an alpha version with Wine 7, it seems to work fine for Stars, but I'm pretty sure a lot of games that work on my previous release are currently broken in this alpha.  Wine 6/7 has so many changes.

http://boxedwine.org/v/BoxedWineVulkanAlphaWin64.zip

Its good news that its fixed now, but it could be quite a while until I have an official release with Wine 6 or 7.
